code/data_setup/parse_imagenet_bb.py

```diff
@@ -38,21 +39,22 @@ def parse_file(bbox_dir, filename):
             bounding_box = child.find('bndbox')
             xmin, xmax, ymin, ymax = parse_bb(bounding_box)
 
-            object_lst = [width, height, name, subcategory, xmin, 
+            object_lst = [filename, width, height, name, subcategory, xmin,
```

Comment:
The other changed lines are just my editor doing automatic clean-up. 
This is the only line I meant to change.  Needed filename for subsequent steps.

I also use the convention x0, y0, x1, y1 instead of xmin, ymin, xmax, ymax.  At some point, we should settle on one convention.